src/ui/settings/components/About.tsx

```diff
@@ -11,84 +11,287 @@
  * SOFTWARE.
  */
 import * as React from 'react';
+import { connect } from 'react-redux';
+import { cadLog, isFirefox } from '../../../services/Libs';
+import IconButton from '../../common_components/IconButton';
+
+const styles = {
+  buttonStyle: {
+    height: 'max-content',
+    padding: '0.75em',
+    width: 'max-content',
+  },
+};
 interface OwnProps {
   style?: React.CSSProperties;
 }
-const About: React.FunctionComponent<OwnProps> = (props) => {
-  const { style } = props;
-  return (
-    <div style={style}>
-      <h1>{browser.i18n.getMessage('aboutText')}</h1>
-      <h5>
-        {browser.i18n.getMessage('versionNumberText', ['CAD'])}:
+
+interface StateProps {
+  bName: browserName;
+  cache: CacheMap;
+  platformInfo: browser.runtime.PlatformInfo;
+  settings: MapToSettingObject;
+}
+enum platformOS {
+  mac = 'Mac OS',
+  win = 'Windows',
+  android = 'Android',
+  cros = 'Chrome OS',
+  linux = 'Linux',
+  openbsd = 'Open/FreeBSD',
+}
+
+const settingOrder = [
+  SettingID.ACTIVE_MODE,
+  SettingID.CLEAN_DELAY,
+  SettingID.CLEAN_DISCARDED,
+  SettingID.CLEAN_DOMAIN_CHANGE,
+  SettingID.ENABLE_GREYLIST,
+  SettingID.CLEAN_OPEN_TABS_STARTUP,
+  SettingID.CLEAN_EXPIRED,
+  SettingID.CLEANUP_CACHE,
+  SettingID.CLEANUP_INDEXEDDB,
+  SettingID.CLEANUP_LOCALSTORAGE,
+  SettingID.CLEANUP_PLUGIN_DATA,
+  SettingID.CLEANUP_SERVICE_WORKERS,
+  SettingID.CONTEXTUAL_IDENTITIES,
+  SettingID.CONTEXTUAL_IDENTITIES_AUTOREMOVE,
+  SettingID.STAT_LOGGING,
+  SettingID.NUM_COOKIES_ICON,
+  SettingID.KEEP_DEFAULT_ICON,
+  SettingID.NOTIFY_AUTO,
+  SettingID.NOTIFY_MANUAL,
+  SettingID.NOTIFY_DURATION,
+  SettingID.ENABLE_NEW_POPUP,
+  SettingID.SIZE_POPUP,
+  SettingID.SIZE_SETTING,
+  SettingID.CONTEXT_MENUS,
+  SettingID.DEBUG_MODE,
+];
+
+type AboutProps = OwnProps & StateProps;
+
+class About extends React.Component<AboutProps> {
+  public render() {
+    const { bName, cache, platformInfo, settings, style } = this.props;
+    const settingSlim = settingOrder.map((s) => {
+      const so = settings[s];
+      return `- ${so.name}: ${so.value}`;
+    });
+    return (
+      <div style={style}>
+        <h1>{browser.i18n.getMessage('aboutText')}</h1>
+        <h5>
+          {browser.i18n.getMessage('versionNumberText', ['CAD'])}:
+          <br />
+          <b>{browser.runtime.getManifest().version}</b>
+        </h5>
+        <a href="https://github.com/mrdokenny/Cookie-AutoDelete/issues">
```

Comment:
Link should probably be `https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/issues`

Reply:
fixed in 0aa3967.  the old one still redirects to it anyway.